source/guides/core-concepts/cypress-studio.md

```diff
@@ -0,0 +1,208 @@
+---
+title: Cypress Studio
+---
+
+{% note info %}
+# {% fa fa-graduation-cap %} What you'll learn
+
+- How to extend tests interactively using the Cypress Studio
+- How to add new tests interactively using the Cypress Studio
+
+{% endnote %}
+
+# Overview
+
+Cypress Studio provides a visual way to generate tests within the Test Runner, by *recording interactions* against the application under test.
+
+The {% url `.click()` click %}, {% url `.dblclick()` dblclick %}, {% url `.type()` type %}, {% url `.check()` check %}{% url `.uncheck()` uncheck %} and {% url `.select()` select %} Cypress commands are supported and will generate test code when interacting with the DOM inside of the Cypress Studio.
```

Comment:
`dblclick()` was actually excluded from this initial release, will be re-added in a future release - otherwise looks good to me

Reply:
Fixed.